.github/workflows/run-build.yaml

```diff
@@ -0,0 +1,254 @@
+          See the list of environment variables that are auto-forwarded in
+          the Nextstrain CLI repo (https://github.com/nextstrain/cli/blob/master/nextstrain/cli/hostenv.py)
+
+          The environment variables must be passed via the `env` input or
+          via GitHub Action's secrets that are inherited by using `secrets: inherit`.
+          The environment variable names should be separated by a space,
+          e.g. "SLACK_TOKEN SLACK_CHANNELS".
+        type: string
+        required: false
+
+      cpus:
+        description: >-
+          Number of CPUs/cores/threads/jobs to utilize at once.
+          Will be passed to the `--cpus` option for `nextstrain build` and the
+          `--cores` option for Snakemake. See `nextstrain build` docs for more details:
+          https://docs.nextstrain.org/projects/cli/page/commands/build/
+        type: number
+        default: 2
+        required: false
+
+      memory:
+        description: >-
+          Amount of memory to make available to the build. Passed to the
+          `--memory` option for `nextstrain build`.
+          See `nextstrain build` docs for more details:
+          https://docs.nextstrain.org/projects/cli/page/commands/build/
+        type: string
+        required: false
+
+      build-args:
+        description: >-
+          Additional command-line arguments to pass to `nextstrain build` after
+          the build directory (i.e. additional args for Snakemake).
+        type: string
+        default: ""
+        required: false
+
+      artifact-name:
+        description: >-
+          Name to use for the build output artifact uploaded at end of the workflow.
+
+          If you're invoking this workflow multiple times from the same calling
+          workflow, you should set this. Otherwise, the default "build-outputs"
+          is probably fine.
+        type: string
+        default: build-outputs
+        required: false
+
+      build-output-paths:
+        description: >-
+          List of build output paths to include in the build output artifact uploaded
+          at the end of the workflow, as a string following the format of the
+          `paths` input of the `actions/upload-artifact` action.
+          For example:
+
+            with:
+              build-output-paths: |
+                results/
+                auspice/
+                logs/
+
+          The output will always include a "build.log" file which contains log
+          messages from the `nextstrain build` command. The envdir created to
+          pass environment variables to the build runtime will always be
+          excluded from the output to prevent secrets from being leaked.
+
+          This is not supported for builds on AWS Batch because the workflow
+          detaches from the build. Please use the `nextstrain build` command
+          locally to reattach to AWS Batch builds to download outputs.
+        type: string
+        default: ""
```

Comment:
Ah, I didn't want to set a default for paths that may not exist in the build. But I guess it doesn't hurt since [the action defaults to `warn` if no files are found](https://github.com/actions/upload-artifact#customization-if-no-files-are-found).  I'll add the `if-no-files-found: warn` input to the upload action to be more explicit about this behavior.

Reply:
Updated in https://github.com/nextstrain/.github/pull/44/commits/135e1cd03ee51b7d6f64ae63eca389944d2dde35